tests/test_feature_selection.py

```diff
@@ -0,0 +1,101 @@
+# Authors: Soledad Galli <solegalli1@gmail.com>
+# License: BSD 3 clause
+import pytest
+import pandas as pd
+
+from feature_engine.feature_selection import DropFeatures
+
+
+def test_FeatureEliminator_initial(dataframe_vartypes, dataframe_na):
+    # test case 1: with dataframe_vartypes
```

Comment:
def test_DropFeatures_drop_2_variables(dataframe_vartypes)

could we rename test to something that indicates what the test is testing? like this example above?

Like this, we also do not need the comment on line 10, which does not explain really clearly what is testing

also, I don't think you need dataframe_na here